python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -231,8 +231,8 @@ class RandomForestClassifier(Base):
 
     Parameters
     -----------
-    n_estimators : int (default = 10)
-        Number of trees in the forest.
+    n_estimators : int (default = 100)
+        Number of trees in the forest. (Default changed to 100 in cuML 0.11)
```

Comment:
we need to change the n_estimators value in def `__init__` func as well.

Reply:
updated, thanks